it/repositories/EventsRepositorySpec.scala

```diff
@@ -53,6 +63,18 @@ class EventsRepositorySpec extends MongoRepositorySpec {
       result.head.venue mustBe VenueType.NEWCASTLE_LONGBENTON.toString
     }
 
+    "filter ALL_EVENTS in LONDON_FSAC" in {
+      repository.save(events).futureValue
+      val result = repository.fetchEvents(EventType.ALL_EVENTS, VenueType.LONDON_FSAC).futureValue
+      result.size mustBe 3
```

Comment:
For these kind of tests can you validate that the data retrieved contains the expected attributes?